src/dmd/e2ir.d

```diff
@@ -188,6 +188,7 @@ private elem *callfunc(const ref Loc loc,
         /* Convert arguments[] to elems[] in left-to-right order
          */
         const n = arguments.dim;
+        size_t nparams = Parameter.dim(tf.parameters);
```

Comment:
This code motion seems unnecessary

Reply:
It isn't necessary but I felt it didn't belong inside the loop either so I moved it out.

Reply:
Fair enough, Walter might have different opinions though.